games-util/pyfa/pyfa-2.15.1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_6 )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit desktop eutils python-single-r1 xdg-utils
+
+DESCRIPTION="Python Fitting Assistant - a ship fitting application for EVE Online"
+HOMEPAGE="https://github.com/pyfa-org/Pyfa"
+
+LICENSE="GPL-3+ all-rights-reserved"
+SLOT="0"
+if [[ ${PV} = 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/pyfa-org/Pyfa.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/pyfa-org/Pyfa/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S=${WORKDIR}/Pyfa-${PV}
+fi
+IUSE="+graph"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RESTRICT="mirror bindist"
+
+DEPEND="
+	>=dev-python/cryptography-2.3[${PYTHON_USEDEP}]
+	>=dev-python/logbook-1.0.0[${PYTHON_USEDEP}]
+	>=dev-python/pyyaml-5.1[${PYTHON_USEDEP}]
+	>=dev-python/sqlalchemy-1.3.0[${PYTHON_USEDEP}]
+	>=dev-python/wxpython-4.0.4[webkit,${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}
+	>=dev-python/beautifulsoup-4.6.0[${PYTHON_USEDEP}]
+	>=dev-python/markdown2-2.3.0[${PYTHON_USEDEP}]
+	>=dev-python/packaging-16.8[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.0.0[${PYTHON_USEDEP}]
+	>=dev-python/roman-2.0.0[${PYTHON_USEDEP}]
+	graph? (
```

Comment:
Also I don't see this USe flag having any function in the ebuild. If this ebuild takes long time to build, you shouldn't use a USE flag to control these deps. 
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs

Reply:
I see you call `python_domodule eos gui service utils graphs` so is it really optional, or should be depended on always?

Reply:
I have to look into this, but you are probably right and this use flag isn't really doing anything anymore.

Reply:
Should you/I update it to always depend on these packages unconditionally, and remove the IUSE? Thanks for figuring the license thing, but I'd like this solved before pushing too.

Reply:
I removed the useflag and also moved the python_deps to the build dependencies instead of just rdep. Also updated the live ebuild with the changes.